lib/sentry/client.ex

```diff
@@ -139,4 +157,17 @@ defmodule Sentry.Client do
     |> Kernel.round()
     |> :timer.sleep()
   end
+
+  defp maybe_call_pre_send_event_function(event) do
+    case Application.get_env(:sentry, :pre_event_send_function) do
+      function when is_function(function) ->
```

Comment:
To ensure `function.(event)` doesn't get a `BadArityError`, you may want to use `is_function(function, 1)`.